images/build/installDotNetCore.sh

```diff
@@ -35,13 +35,6 @@ mkdir -p $DOTNET_DIR
 tar -xzf dotnet.tar.gz -C $DOTNET_DIR
 rm dotnet.tar.gz
 
-# Create a link : major.minor => major.minor.path
```

Comment:
We will no longer create links of this sort and will reply on semantic versioning to handle it. However, we will still have links like `lts` or `stable`